packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditReducers.ts

```diff
@@ -79,6 +86,31 @@ export function applySynchronizationOp(state: DDSFuzzTestState<SharedTreeFactory
 		}
 	}
 }
+function generateLeafNodeSchemas(nodeTypes: string[]) {
```

Comment:
I suspect there's a more ergonomic way to do this using libraries, but it's not a big deal.

Reply:
Left as is for now, but added a TODO comment mentioning this